src/main/java/com/example/ecommerce/controller/MyOrderController.java

```diff
@@ -0,0 +1,14 @@
+package com.example.ecommerce.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+public class MyOrderController {
+        @RequestMapping("/myOrders")
```

Comment:
```suggestion
        @RequestMapping("/my-orders")
```
url thì em cso thể dùng snake case thay vì camel case  nhé